### ISSUE TITLE
Add Muzzey 2014 Candida albicans data

### ISSUE THREAD

**Issue by FlicAnderson** [riboviz/example-datasets#19]
More details via [riboviz/#29](https://github.com/riboviz/riboviz/issues/29):

Add in the Muzzey et al 2014 dataset (M-Ca_2014) to `example-datasets` repo: 

https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52236

Muzzey D, Sherlock G, Weissman JS. Extensive and coordinated control of allele-specific expression by both transcription and translation in Candida albicans. Genome Res 2014 Jun;24(6):963-73. PMID: 24732588

**Comment by FlicAnderson** [comment]
I've updated this branch `fungi-muzzey-2014-candida`: 
* merged in a recent `master`, 
* added dataset to the kingdom README
* renamed annotation & contaminants files with full scientific name for organism
* updated provenance.txt files to reflect new annotation/contaminant file names
* renamed yaml to reflect new transcriptome name format, number of samples & removed species name to follow examples in `saccharomyces`
* upgraded yaml using `upgrade_config_file` tool, then manually moved the provenance block back up to the top again :smile_cat: 
* ready to try running it on Eddie. 

Will try the job submission script generator for this...
Via branch ` py-create-job-submission-script-228`
```
# create submission file
(riboviz) fanders6@ceres:~/riboviz/riboviz$ python -m riboviz.tools.create_job_script -i jobs/eddie-template.sh -o run_M-Ca_2014.sh --config-file ~/riboviz/example-datasets/fungi/candida/Muzzey_2014_RPF_3-samples_CDS_with_120utrs_config.yaml --r-libs /exports/csce/eddie/biology/groups/wallace_rna/Rlibrary --job-name M-Ca_2014 --job-runtime 24:00:00 --job-email <EMAIL>

# check contents
(riboviz) fanders6@ceres:~/riboviz/riboviz$ cat run_M-Ca_2014.sh 
#!/bin/sh
# Grid Engine options (lines prefixed with #$)
# Job name:
#$ -N M-Ca_2014
# Use the current working directory:
#$ -cwd
# Runtime limit:
#$ -l h_rt=24:00:00
# RAM:
#$ -l h_vmem=8G
# Request parallel environment and number of CPUs:
#$ -pe mpi 4
# Redirected output file name format:
#$ -o $JOB_NAME-$JOB_ID-$HOSTNAME.o
# Redirected error file name format:
#$ -e $JOB_NAME-$JOB_ID-$HOSTNAME.e
# E-mail address for batch job events:
#$ -M <EMAIL>
# Events triggering emails about batch job.
# Any combination of b - begin, e - end, a - abort, s - suspend:
#$ -m beas

# Initialise the environment modules.
. /etc/profile.d/modules.sh

#!/usr/bin/env bash
export R_LIBS=/exports/csce/eddie/biology/groups/wallace_rna/Rlibrary
module load openmpi
module load igmm/apps/BEDTools
module load igmm/apps/bowtie
module load igmm/apps/hdf5
module load igmm/apps/HISAT2
module load igmm/apps/pigz
module load igmm/apps/R/3.6.3
module load igmm/apps/sratoolkit/2.10.8
module load anaconda
source activate riboviz

echo "Running Nextflow riboviz..."

nextflow run prep_riboviz.nf -params-file /home/fanders6/riboviz/example-datasets/fungi/candida/Muzzey_2014_RPF_3-samples_CDS_with_120utrs_config.yaml -work-dir work -ansi-log false -with-report nextflow-report.html  

echo "Nextflow riboviz run complete!"
```

* copied this from local to Eddie (couldn't swap riboviz branch on Eddie as I've got mouse data running on `develop`)

* created dataset folder on Eddie
* downloading SRA files via prefetch & fasterq-dump (currently running)

NEXT STEPS: 
Will set up files ready to run after the SRA downloads are complete. This involves: 
 - swapping branch of example-datasets to `fungi-muzzey-2014-candida`
 - copy yaml out to symlinked (to scratch) dataset folder 
 - check paths in yaml & job submission script 
 - check the validate-only results (might do this interactively)
 - submit the dataset run via the job submission script.

**Comment by FlicAnderson** [comment]
I actually ran into errors when it came to submitting using the job submission scripts generated by #228 code, so ended up using a variation on my usual job submission script process and dataset setup.  I'll update issue #228 with more details on what seemed to go wrong. 

The run completed for all samples, outputs look pretty good, and `master` has been merged into `fungi-muzzey-2014-candida` ahead of a PR. 

Since this is an old dataset we've already done QC for and approved, I've not attempted running `check_fasta_gff` on the annotation files.  The current outputs match the outputs I got in 2019, so I'm happy with this.
